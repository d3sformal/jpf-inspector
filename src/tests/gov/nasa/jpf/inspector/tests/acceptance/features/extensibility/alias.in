# The following two commands should do the same thing:
testAlias
hello