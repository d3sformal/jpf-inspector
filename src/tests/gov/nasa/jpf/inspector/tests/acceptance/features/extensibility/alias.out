Hello to you, too, dear user! The JPF Inspector command loop is working!
Hello to you, too, dear user! The JPF Inspector command loop is working!